torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -213,6 +213,19 @@ RegisterOperators reg(
            push(stack, autograd::make_variable(at::scalar_to_tensor(b)));
            return 0;
          }),
+     // FIXME: It would be nice to remove this if symbolic_script can get by without
+     Operator(
+         "aten::_float(Scalar a) -> float",
```

Comment:
Why does `prim::Float(Scalar a) -> float` not work ?

Reply:
I needed something to use in symbolic_script, so I'm not sure I have prim ops there. But I've now changed the Scalar<->float conversion to be a bit more prudent about Scalar not always being float.

Reply:
i think the prim ops work - try using a cast - `float(exponent`

Reply:
@t-vi I feel `float` already works given we register `prim::Float` as `float` here. https://github.com/pytorch/pytorch/blob/ada10ad416f91602d46797b50b3faebcacc6e767/torch/csrc/jit/script/compiler.cpp#L390

Reply:
It got eliminated before (because the input claimed it was float). But number fixes it. :)